topo/node/ceos/ceos.go

```diff
@@ -81,17 +76,14 @@ func (n *Node) WaitCLIReady() error {
 
 // PatchCLIConnOpen sets the OpenCmd and ExecCmd of system transport to work with `kubectl exec` terminal.
 func (n *Node) PatchCLIConnOpen(ns string) error {
-	switch t := n.cliConn.Transport.(type) {
-	case *scraplitest.TestingTransport:
-		t.ExecCmd = "kubectl"
-		t.OpenCmd = []string{"exec", "-it", "-n", ns, n.pb.Name, "--", "Cli"}
-	case *scraplitransport.System:
-		t.ExecCmd = "kubectl"
-		t.OpenCmd = []string{"exec", "-it", "-n", ns, n.pb.Name, "--", "Cli"}
-	default:
+	t, ok := interface{}(n.cliConn.Transport.Impl).(scraplitransport.SystemTransport)
```

Comment:
can it be
```go
t, ok := n.cliConn.Transport.Impl.(scraplitransport.SystemTransport)
```
?

Reply:
Good call... was wrapped around the axle there for a minute -- fixed 😁